src/blockchain-wallet.js

```diff
@@ -854,3 +854,15 @@ Wallet.prototype.loadExternal = function () {
     return this._external.fetch();
   }
 };
+
+Wallet.prototype.incStats = function () {
+  var balanceNoWO = function (a) {
+    return a.isWatchOnly ? 0 : a.balance;
+  }
+  var legacyBalance = this.keys.map(balanceNoWO).reduce(Helpers.add, 0);
+  var mvBool = this.hdwallet ? this.hdwallet.isMnemonicVerified : false;
+  API.incrementSecPassStats(this.isDoubleEncrypted);
+  API.incrementRecoveryStats(mvBool);
+  API.incrementLegacyUseStats(legacyBalance > 0);
```

Comment:
`incrementLegacyUseStats` only sends a boolean, not the balance. So there would be a lot of false positives for how many people use legacy.